Source/WebCore/inspector/InspectorCanvas.h

```diff
@@ -130,7 +130,7 @@ class InspectorCanvas final : public RefCounted<InspectorCanvas> {
 
     String m_identifier;
 
-    CanvasRenderingContext& m_context;
+    WeakPtr<CanvasRenderingContext> m_context;
```

Comment:
Need `RefPtr<> protectedContext` and possibly `nullptr` checks where `m_context` is used in the class.